### PR TITLE
update broken link:https://github.com/kubernetes/community/blob/maste…

### PR DESCRIPTION
**Pull request by lovebaby979**
…r/keps/sig-cloud-provider/0019-cloud-provider-documentation.md

### CODE REVIEW

**Comment by onlydole** [comment]
Thanks for your contribution, @lovebaby979!

File: communication/meeting-notes-archive/201808-201902_Community_Meeting_Minutes.md [hunk 1]
```diff
@@ -2274,7 +2274,7 @@ Also @jorge for all the live stream work! You two rock
 *   [ 0:00 ] <strong>KEP o' the Week </strong>[Chris Hoge, @hogepodge, on behalf of Nishi Davidson, @d-nishi]
     *   Part of SIG Cloud Provider
         *   Coordinates stuff among all cloud providers
-    *   [https://github.com/kubernetes/community/blob/master/keps/sig-cloud-provider/0019-cloud-provider-documentation.md](https://github.com/kubernetes/community/blob/master/keps/sig-cloud-provider/0019-cloud-provider-documentation.md)  - Accepted 
+    *   [https://github.com/kubernetes/community/blob/master/sig-cloud-provider/CHARTER.md](https://github.com/kubernetes/community/blob/master/sig-cloud-provider/CHARTER.md)  - Accepted
```
Comment (mrbobbytables) [inline communication/meeting-notes-archive/201808-201902_Community_Meeting_Minutes.md]:
```suggestion
    *   [https://github.com/kubernetes/enhancements/blob/master/keps/sig-cloud-provider/20180731-cloud-provider-docs.md](https://github.com/kubernetes/enhancements/blob/master/keps/sig-cloud-provider/20180731-cloud-provider-docs.md)  - Accepted 
```
KEPs were relocated to the [kubernetes/enhancements](https://github.com/kubernetes/enhancements/) repo late last year along with adopting a date-prefix. 👍 

If you could, please update both instances to reference the KEP link in the suggestion.

**Comment by parispittman** [comment]
hi @lovebaby979 - looks like you have one comment left from bob to address and then we can merge! :)

**Comment by mrbobbytables** [comment]
As there hasn't been any updates, I'm going to go ahead and close this for now. Please reopen or create a new PR if you'd like to tackle this later 👍 

Thanks!

/close

**Pull request closed without merging**